packages/liveblocks-core/src/__tests__/room.test.ts

```diff
@@ -190,20 +213,20 @@ describe("room / auth", () => {
   });
 
   test("private authentication that does not returns json should throw", async () => {
-    const room = createRoom(
+    const controlledRoom = createTestableRoom(
```

Comment:
Renamed `room` to `controlledRoom` here, to make it more obvious that this isn't a `Room` instance.